### PR TITLE
docs: Add a sample config for autopush and fix some settings

### PR DESCRIPTION
**Pull request by Mcat12**
- Removed unused `debug` setting
- Added default for `endpoint_hostname`
- Fixed invalid default for `crypto_key`

Related to #10

### CODE REVIEW

File: autopush/src/settings.rs [hunk 1]
```diff
@@ -73,8 +72,9 @@ impl Settings {
         s.set_default("max_connections", 0)?;
         s.set_default("close_handshake_timeout", 0)?;
         s.set_default("endpoint_scheme", "http")?;
+        s.set_default("endpoint_hostname", "localhost")?;
         s.set_default("endpoint_port", 8082)?;
-        s.set_default("crypto_key", Fernet::generate_key())?;
```
Comment (jrconlin) [inline autopush/src/settings.rs:77]: I know that we're switching to a CSV list for this value.
Don't we reject any string that has "[]" as a wrapper in `ServerOptions::from_settings`?

File: configs/autopush.toml.sample [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+# The name of the router table
+#router_tablename = "router"
+
+# The prefix of the message table(s)
+#message_tablename = "message"
+
+# A (stringified) list of Fernet keys to use when encrypting the notification
+# endpoint URL. The default is a single auto-generated key.
+#crypto_key = "[]"
```
Comment (jrconlin) [inline configs/autopush.toml.sample:66]: remove the brackets, or use some pre-generated test value (noting that it should not be used in production in the comment?)

File: autopush/src/settings.rs (hunk 1 above)
Comment (Mcat12) [reply autopush/src/settings.rs:77]: We reject the opposite: if the string does not have a `[]` wrapper.

Comment (jrconlin) [reply autopush/src/settings.rs:77]: Ah! you're right. I missed the leading `!`.

**Pull request merged**